lib/puppet/defaults.rb

```diff
@@ -742,14 +742,15 @@ def self.default_basemodulepath
 
         Defaults to the node's fully qualified domain name.",
       :hook => proc { |value| raise(ArgumentError, _("Certificate names must be lower case")) unless value == value.downcase }},
-    :dns_alt_names => {
-      :default => '',
+    :subject_alt_names => {
```

Comment:
Unfortunately, we can't change setting names in a .z. Note each setting is automagically made available as a command line argument, so this could break anyone calling puppet programmatically with the old command line argument, which would now be unrecognized.

Reply:
Nevermind, I see the `dns_alt_names` setting below.